config/locales/en/course/assessment/submission/submissions.yml

```diff
@@ -39,6 +39,7 @@ en:
             auto_grade: 'Auto Grade Submission'
             evaluate_answers: 'Evaluate Answers'
             unsubmit: 'Unsubmit Submission'
+            unmark: 'Ungrade Submission'
```

Comment:
To avoid confusion I might suggest to be more explicit, something like `Revert to Submitted`.